apps/api/src/index.ts

```diff
@@ -106,14 +106,14 @@ async function initAPI() {
     // #endregion
 
     // #region Start Server
-    server.listen(settings.port, () => {
-        console.log(`Server listening on port ${settings.port}`);
+    server.listen(settings?.port, () => {
+        console.log(`Server listening on port ${settings?.port}`);
         openInBrowser();
```

Comment:
```suggestion
```
This can be removed, it's no longer used.

Reply:
the ?. does not really mean anything, its just to make typescript happy. it has to be there because the settings variable can be null